main/ipc.js

```diff
@@ -12,7 +12,8 @@ const ipcMainEvents = Object.freeze({
   JOB_STATS_UPDATED: 'station:job-stats-updated',
 
   UPDATE_CHECK_STARTED: 'station:update-check:started',
-  UPDATE_CHECK_FINISHED: 'station:update-check:finished'
+  UPDATE_CHECK_FINISHED: 'station:update-check:finished',
+  UPDATE_AVAILABLE: 'station:update-available'
```

Comment:
I am afraid it's not enough to emit an IPC event.

- The Station starts running in the Tray only with the main window hidden.
- When the main window is hidden, it does not receive events.
- As a result, the new banner is not shown.

I am not sure why our frontend code is not receiving the event. I believe we do forward IPC events to the window. Maybe when the window was not shown then our frontend code installing event handlers did not run?

I think we need to add a new method allowing the frontend to query the status of the update check.

I am going to look into this myself right now and let you know what I find.

Reply:
Done in 8de84e0227b23d525e8f7deb2a5ba21c1aa87ce6

Reply:
Interesting, let's keep an eye on this issue in the future 🤔